doc/lua_api.txt

```diff
@@ -3364,6 +3364,23 @@ It can be created via `Settings(filename)`.
     * write changes to file
 * `to_table()`: returns `{[key1]=value1,...}`
 
+### `Raycast`
```

Comment:
This should be minetest.raycast. It's not a type as such, even though it returns a type
and the iter minetest.raycast_iter

Is it possible to support both with one object, I wonder?